TACC/protx-dashboard
# Use env variables to pin database versions

## Overview: ##

A PR to pin databases so we are aware what source code goes with what database versions.  

Users would need to place any new databases and then update the .env file.  

**if we like this**, then we need to add these env variables to Core-Deployments 's protx docker-compose files 

In this PR:
* the database versions are env variables and so pinned in .env (for docker-compose)
*  some refactoring of where those database paths are defined in our python code (i.e. protx/utils/db.py)

## Related Jira tickets: ##

None

## Testing Steps: ##
0. Ensure you have the pinned databases in your ~/protx-data folder
1. docker exec -it protx python3 scripts/run_queries.py --clear-cache
2. Run unit tests
3. Try app
